src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -267,6 +253,12 @@ internal static void ConfigureApplicationInsights(HostBuilderContext context, IL
                             break;
                         }
                     }
+
+                    // Disable auto-http tracking when in placeholder mode.
+                    builder.Services.Configure<ApplicationInsightsLoggerOptions>(o =>
+                    {
+                        o.HttpAutoCollectionOptions.EnableHttpTriggerExtendedInfoCollection = false;
```

Comment:
@safihamid FYI. I added this section here -- this will disable auto-HTTP requests while in placeholder mode. This is very similar to the way you're removing the DependencyTracking module above -- but we'd added a setting to handle it.